example/app/lib/main.dart

```diff
@@ -74,6 +74,24 @@ class _MyHomePageState extends State<MyHomePage> {
 
               return new Mutation(
                 mutations.addStar,
+                onCompleted: (Map<String, dynamic> data) {
+                  showDialog(
```

Comment:
Can you move the `onCompleted` below the `builder argument, so it matches the order in the class constructer.